### ISSUE TITLE
[SF][A11Y][Toolbar]: Provide interface to customize accessible name of overflow button

### ISSUE THREAD

**Issue by josepinesu** [SAP/ui5-webcomponents#11968]
### Is your feature request related to a problem?

An a11y issue was raised to the toolbar button: 'Additional Options' buttons are missing descriptive accessible names. 
https://stackblitz.com/edit/github-xjv9w2jh-hecupc6r?file=src%2FApp.tsx

<img width="1380" height="706" alt="Image" src="https://github.com/user-attachments/assets/a487826c-9108-4310-9529-f64e66f770a7" />

### Describe the solution you'd like

Provide interface to customize accessible name for overflow button, so that we can replace "Additional Options" to "More actions for 'opportunity name'".

### Declaration

- [ ] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by NHristov-sap** [comment]
Hello @SAP/ui5-webcomponents-topic-p, 

Please take a look of this feature request.

Best Regards,
Nikolay Hristov

**Comment by judith9209** [comment]
Hi, would you let us know how you would like to continue with this request? Many thanks in advance!

**Comment by judith9209** [comment]
Hi @NHristov-sap , would you share the related jira item?

**Comment by petyabegovska** [comment]
BGSOFUIPIRIN-7044
